addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/VoiceRSSTTSService.java

```diff
@@ -74,6 +77,7 @@ protected void activate(Map<String, Object> config) {
         }
     }
 
+    @Modified
```

Comment:
@wborn Is it possible that you also need `@Override` on this?

Reply:
I can reproduce the issue and it has got to do with the generated annotations so I'll have a look again what's wrong. ;-)